geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/ShowMissingDiskStoresFunction.java

```diff
@@ -65,23 +65,25 @@ public void execute(FunctionContext context) {
           }
         }
       }
+    } catch (CacheClosedException ignored) {
```

Comment:
I don't think we want to do this specially for ShowMissingDiskStoresFunction alone. It's ok for the function to throw this exception and have the function callers catch that. We will need to adjust the test to assert getting this exception for a member without cache.

Reply:
My original thought was to preserve the original result, i.e. return an empty result if the cache is null. However, you're right, we already have a mechanism to handle unexpected exceptions so I've removed the extra catch clause.

Test was adjusted accordingly. I've also removed redundant fail() assertions from the tests since use of fail() does not follow our testing best practices.

Reply:
Precheckin ran green except for known issues from unrelated commit 44e8432e6637afbde478420c26e260630a922a5e